packages/actionmenu/package.json

```diff
@@ -14,7 +14,6 @@
     "build:css": "build-css --useGlamor -i dist/cjs/css/index.js",
     "build:esm": "tsc --project tsconfig.build.json --module es2015 --target es5 --outDir dist/esm",
     "build:watch": "yarn build:esm -- --watch",
-    "prepublish": "yarn build",
```

Comment:
`prepublish` is deprecate. i think we're supposed to use `prepare` or `prepublishOnly`. i just decided to remove these and build once at the root